Web_Development/Javascript/Topics/5.Syntax/Objects/README.md

```diff
@@ -0,0 +1,136 @@
+
+__Example:__
+```javascript
+const officer = {
+    firstName : "Glenn",
+    lastName : "Wilson",
+    age : 28
+};
+```
+In the above example,
+1. ```officer``` is an object.
+2. ```firstName```, ```lastName```, ```age``` are the keys.
+3. ```"Glen"```, ```"Wilson"```, ```28``` are the values.
+# Object Properties
+Properties are nothing but the values associated with the JavaScript object.
+
+## Syntax for Accessing JavaScript Properties
+```js
+objectName.property // officer.age
+```
+or
+```js
+objectName["property"] // officer["age"]
+```
+You can also add new properties.
+
+__Example:__
+```javascript
+officer.country = "American"; // adds country to officer object
+```
+You can also delete the properties by using ```delete``` keyword.
+
+__Example:__
+```javascript
+delete officer.country; // deletes country from officer object
+```
+# Object Methods
+Object method are the functions that can be performed on any object.
+## Syntax for accessing Object methods
+
+```js
+objectName.methodName()
+```
+__Example:__
+```javascript 
+name = officer.fullName();
+```
+We can also use ```this``` keyword.
+
+```this``` refers to owner of the function. Like in the below example ```this.firstName``` means ```firstName``` property of the ```officer``` object.
+
+__Example:__
+```javascript
+const officer = {
+    firstName : "Glenn",
+    lastName : "Wilson",
+    age : 28,
+    sent : function() {
+        return this.firstName + " " + "is" + officer.age + "years old."; // Glenn is 28 years old.
+    }
+};
+```
+
+# Getters and Setters
+Getters and Setters are the JavaScript Accessors that allows you to define the Object Accessors.
+## Getter
+This makes use of ```get``` keyword. It works when a ```function``` without arguments is read.
+
+__Example:__
+```javascript
+const officer = {
+    firstName : "Glenn",
+    lastName : "Wilson",
+    age : 28,
+
+    get personAge(){
+        return `${this.age}`;
+    }
+};
+alert(officer.personAge); //28
+```
+
+The above example uses the ```personAge``` to get the value of ```age``` property.
+
+## Setter
+This makes use of ```set``` keyword. It works when a ```function``` with one argument is called when the property is set.
+
+__Example:__
+```javascript
+const officer = {
+    firstName : "Glenn",
+    lastName : "Wilson",
+    age : 28,
+    nationality : "",
+
+    set nation(nation){
+        this.nationality = nation;
+    }
+};
+officer.nation = "American"; // set an object property
+alert(officer.nationality); // American
+```
+The above examples uses ```nation``` to set the value of ```nationality``` property.
+
+# Difference between Getters and Setters
+The basic difference between them is that __Getters__ are the object accessor that is used to get the property from an object, whereas __Setters__ are the object accessor that is used to set the property of an object.
+
+# Conclusion
+
+Congratulations! You've successfully learnt about JavaScript Object.
+
+Keeping exploring JavaScript :wave:
```

Comment:
Little grammatiacal correction is require **Keep Exploring JavaScript**. Hope you get it:-)

Reply:
Ma'am, I've corrected it.